docs/apm/storage-explorer.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[storage-explorer]]
+=== Storage explorer
+
+beta::[]
+
+Analyze your APM data and manage costs with **storage explorer**.
+For example, analyze the storage footprint of each of your services to see which are producing
+large amounts of data--then change the sample rate of a service to lower the amount of data ingested.
+Or, expand the time filter to visualize data trends over time so that you can better forecast
+and prepare for future storage needs.
+
+[role="screenshot"]
+image::apm/images/storage-explorer-overview.png[APM Storage explorer]
+
+[float]
+==== Index lifecycle phases
+
+A default {apm-guide-ref}/ilm-how-to.html[index lifecycle policy] is applied to each APM data stream,
+but can be customized depending on your business needs.
+Use the **Index lifecycle phase** dropdown to visualize and analyze your storage by phase.
+
+Customizing the default APM index lifecycle policies can save money by specifying things like:
+
+* The point at which an index can be moved to less performant hardware.
+* The point at which availability is not as critical and the number of replicas can be reduced.
+* When the index can be safely deleted.
+
+See {apm-guide-ref}/ilm-how-to.html[Index lifecycle management] to learn more about customizing
+the default APM index lifecycle policies.
+
+[float]
+==== Service statistics
+
+The service statistics chart provides detailed information on each service:
```

Comment:
The services size chart provides the estimated size for each service over time. The service statistics table provides detailed information on each service.

Reply:
Yes, the each dropdown from the service table shows a detailed break-down of the selected service, so you can get into the details of which type of metrics is taking the most storage space for example.

Reply:
👍 I'll add a small section on the size chart.